### ISSUE TITLE
Improve way based point of interest processing

### ISSUE THREAD

**Issue by vmarc** [vmarc/knooppuntnet#325]
Lately there have been a lot of changes in OpenStreetMap, first removing _"name:ru"_ tags, and later reverts to undo this removals (vandalism, see forum). This resulted in unusual large minute diff files and changes that take a long time to process in knooppuntnet.

More specifically the problem is in processing large amounts of way based point of interest (poi) updates.  For each such poi, a request is done to the local overpass database to determine the center of the poi.  For node based pois the location is included in the minute diff itself, but for ways this is not the case.

As an example, while reproducing the problem in the development environment, the pois in minute diff 005/719/419 take 14 minutes to process. 14 minutes to process 1 minute worth of changes is not good.

**Comment by vmarc** [comment]
The example minute diff (005/719/419) contains 65 changesets with updates to 52594 elements:
- 41531 created or modified nodes
- 10825 created or modified ways
- 68 created or modified relations
- 170 deletions

Created or modified elements considered point of interest (poi) by knooppuntnet:
- 26066 poi nodes
- 4717 poi ways
- 37 poi relations

Number of pois within the [areas supported by knooppuntnet](https://knooppuntnet.nl/en/poi/areas):
- 1 way

In other words: all the poi related processing for this minute diff results in a single knooppuntnet database poi update only!

**Comment by vmarc** [comment]
Implemented in release 4.0.45.

Performance is improved by reducing the number of roundtrips to the overpass database by fetching the center of ways and relations in batch instead of one by one. The test example that did take 14 minutes in the development environment now takes 45 seconds. Production is faster.
